### ISSUE TITLE
Research Page Bug

### ISSUE THREAD

**Issue by smaranjitghose** [smaranjitghose/awesome-portfolio-websites#742]
![Reserach Page UI Fix](https://user-images.githubusercontent.com/46641503/122481831-0ef3f580-cfed-11eb-8f29-a4dc3a960380.png)

# Requirements:

- Whenever one clicks over the Bibtex (would be changed to Cite in #741 ) or Abstract Button, the size of the card should not abruptly change. To fix this, display the text as a pop up with a X button to close it
- Both Abstract and Bibtex Content should not be visible at the same time. When one is being displayed the other has to be collapsed/closed.

**Comment by tauqeerali1** [comment]
Please assign this issue to me.

**Comment by deeqakkk** [comment]
@tauqeerali1 you can work on it.

**Comment by tauqeerali1** [comment]
I have doubt in 1st point when we click popup X button that will close Cite/Abstract button also or this is only a simple popup?

**Comment by tauqeerali1** [comment]
I have doubt in 1st point when we will click popup "X" button that will close Cite/Abstract button also or this is only a simple popup?

**Comment by tauqeerali1** [comment]
please explain me @smaranjitghose

**Comment by smaranjitghose** [comment]
@tauqeerali1  The position of the cite/abstract buttons should be the same. No tweaking with that.

Now, when one clicks on the cite/abstract button, the text corresponding to that is fetched from the research.js and displayed in a pop up window (smaller size obviously, perhaps 1/10th of the total window size) with the necessary title, the X button simply closes that pop up window.  Check out Google Scholar, look up any paper and click on the quotation symbol below the paper information for reference

**Comment by deeqakkk** [comment]
@tauqeerali1 any progress updates on it?

**Comment by tauqeerali1** [comment]
yes i am doing on it now i made popup withh X button and now i am working on button which not show at the same time when we will click on that button

**Comment by deeqakkk** [comment]
@tauqeerali1 try to make a pull request before 1 july.

**Comment by tauqeerali1** [comment]
okay... i will do pull request as soon as posible

**Comment by Ananya-Misra** [comment]
@smaranjitghose Please assign this issue to me.

**Comment by deeqakkk** [comment]
@Ananya-Misra Yes you can work on it, keep us updated!

**Comment by smaranjitghose** [comment]
Any updates @Ananya-Misra ?
